tests/model/test_time_to_triage.py

```diff
@@ -0,0 +1,96 @@
+        assert test_perf_alert_summary.triage_due.isoweekday() == 4
+
+    def test_alert_summary_created_tuesday(self, test_perf_alert_summary):
+        test_perf_alert_summary = copy.deepcopy(test_perf_alert_summary)
+        test_perf_alert_summary.created = datetime.datetime.fromisoformat('2022-05-31')
+        test_perf_alert_summary.triage_due = None
+
+        assert not test_perf_alert_summary.triage_due
+
+        test_perf_alert_summary.time_to_triage()
+        test_perf_alert_summary.save()
+
+        # created tuesday = 2, +3=5
+        assert test_perf_alert_summary.triage_due.isoweekday() == 5
+
+    def test_alert_summary_created_wednesday(self, test_perf_alert_summary):
+        test_perf_alert_summary = copy.deepcopy(test_perf_alert_summary)
+        test_perf_alert_summary.created = datetime.datetime.fromisoformat('2022-06-01')
+        test_perf_alert_summary.triage_due = None
+
+        assert not test_perf_alert_summary.triage_due
+
+        test_perf_alert_summary.time_to_triage()
+        test_perf_alert_summary.save()
+
+        # created wednesday = 3, +3=6 => 1
+        assert test_perf_alert_summary.triage_due.isoweekday() == 1
+
+    def test_alert_summary_created_thursday(self, test_perf_alert_summary):
+        test_perf_alert_summary = copy.deepcopy(test_perf_alert_summary)
+        test_perf_alert_summary.created = datetime.datetime.fromisoformat('2022-06-02')
+        test_perf_alert_summary.triage_due = None
+
+        assert not test_perf_alert_summary.triage_due
+
+        test_perf_alert_summary.time_to_triage()
+        test_perf_alert_summary.save()
+
+        # created thursday = 4, +3=7 => 1+1=2
+        assert test_perf_alert_summary.triage_due.isoweekday() == 2
+
+    def test_alert_summary_created_friday(self, test_perf_alert_summary):
+        test_perf_alert_summary = copy.deepcopy(test_perf_alert_summary)
+        test_perf_alert_summary.created = datetime.datetime.fromisoformat('2022-06-03')
+        test_perf_alert_summary.triage_due = None
+
+        assert not test_perf_alert_summary.triage_due
+
+        test_perf_alert_summary.time_to_triage()
+        test_perf_alert_summary.save()
+
+        # created friday = 5, +3=8 => 1+2=3
+        assert test_perf_alert_summary.triage_due.isoweekday() == 3
+
+    def test_alert_summary_created_saturday(self, test_perf_alert_summary):
+        test_perf_alert_summary = copy.deepcopy(test_perf_alert_summary)
+        test_perf_alert_summary.created = datetime.datetime.fromisoformat('2022-06-04')
+        test_perf_alert_summary.triage_due = None
+
+        assert not test_perf_alert_summary.triage_due
+
+        test_perf_alert_summary.time_to_triage()
+        test_perf_alert_summary.save()
+
+        # created saturday = 6 => 1, +3=4
+        assert test_perf_alert_summary.triage_due.isoweekday() == 4
+
+    def test_alert_summary_created_sunday(self, test_perf_alert_summary):
+        test_perf_alert_summary = copy.deepcopy(test_perf_alert_summary)
```

Comment:
Why do you use `deepcopy` ? I don't think it's necessary, changes done in test will not affect the other tests to my knowledge.

Reply:
as I saw, the tests are running sequentially and if I use the same object, the second test would have the object in the state first test left it. Will try running again.

Reply:
I wonder if that might be the case because you are using a test class? Most of the test modules in Treeherder have just plain test functions. Could you try it that way so you don't need the `deepcopy`?